### PR TITLE
Change instance types endpoint

### PR DESCRIPTION
**Pull request by adiabramovitch**

### CODE REVIEW

File: api/openapi.gen.json [hunk 1]
```diff
@@ -286,6 +250,30 @@
           }
         }
       }
+    },
+    "/sources/{ID}/instance_types": {
+      "get": {
+        "description": "Return a list of instance types",
+        "operationId": "getInstanceTypeList",
```
Comment (adiabramovitch) [inline api/openapi.gen.json:257]: Should I add info about the ID? @ezr-ondrej @lzap

Comment (ezr-ondrej) [reply api/openapi.gen.json:257]: I wonder why it is not generated by itself, but sure, I believe we should have type and description for the ID

File: api/openapi.gen.json [hunk 2]
```diff
@@ -286,6 +250,42 @@
           }
         }
       }
+    },
+    "/sources/{ID}/instance_types": {
```
Comment (ezr-ondrej) [inline api/openapi.gen.json:254]: @akhil-jha @mshriver we're renaming the endpoint, does this makes more sense? Will that be easier to test?

**Review by ezr-ondrej — DISMISSED** [review]
LGTM, but we should gate OpenAPI spec changes by QE ack I believe.

**Review by ezr-ondrej — APPROVED** [review]

**Comment by ezr-ondrej** [comment]
@adiabramovitch I've just now realized: should we have accompanying PR to the frontend?

**Comment by adiabramovitch** [comment]
> @adiabramovitch I've just now realized: should we have accompanying PR to the frontend?

Good one, yes we should.

**Comment by ezr-ondrej** [comment]
> /retest

did it work?

**Comment by ezr-ondrej** [comment]
This is waiting for QE to have a setup to test this, this endpoint was never tested, but given we touch it already, it is good idea to wait with it until we have proper pipeline verifying it :)

**Review by lzap — APPROVED** [review]

**Comment by adiabramovitch** [comment]
@akhil-jha Now that you can test the instance type endpoint, can we approve this change? 😄

**Review by akhil-jha — APPROVED** [review]
Yay! It's working!

**Pull request merged**